<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6719" started-at="2019-11-05T15:28:04Z" finished-at="2019-11-05T15:28:10Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6719" started-at="2019-11-05T15:28:04Z" finished-at="2019-11-05T15:28:10Z">
      <class name="restasure.restaureapi.googleapi">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:restasure.restaureapi.googleapi@66a3ffec]" name="beforeClass" is-config="true" duration-ms="4820" started-at="2019-11-05T15:28:04Z" finished-at="2019-11-05T15:28:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="test_NumberOfCircuitsFor2017Season_ShouldBe20()[pri:0, instance:restasure.restaureapi.googleapi@66a3ffec]" name="test_NumberOfCircuitsFor2017Season_ShouldBe20" duration-ms="1625" started-at="2019-11-05T15:28:08Z" finished-at="2019-11-05T15:28:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_NumberOfCircuitsFor2017Season_ShouldBe20 -->
        <test-method status="FAIL" signature="test_ResponseHeaderData_ShouldBeCorrect()[pri:0, instance:restasure.restaureapi.googleapi@66a3ffec]" name="test_ResponseHeaderData_ShouldBeCorrect" duration-ms="242" started-at="2019-11-05T15:28:10Z" finished-at="2019-11-05T15:28:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[1 expectation failed.
Expected header "Content-Length" was not "4567", was "4551". Headers are:
Date=Tue, 05 Nov 2019 16:12:28 GMT
Server=Apache/2.2.15 (CentOS)
X-Powered-By=PHP/5.3.3
Access-Control-Allow-Origin=*
Content-Length=4551
Connection=close
Content-Type=application/json; charset=utf-8
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected header "Content-Length" was not "4567", was "4551". Headers are:
Date=Tue, 05 Nov 2019 16:12:28 GMT
Server=Apache/2.2.15 (CentOS)
X-Powered-By=PHP/5.3.3
Access-Control-Allow-Origin=*
Content-Length=4551
Connection=close
Content-Type=application/json; charset=utf-8

at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:83)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:60)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:235)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:247)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:471)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:636)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:154)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:166)
at io.restassured.internal.ResponseSpecificationImpl.header(ResponseSpecificationImpl.groovy:177)
at io.restassured.internal.ValidatableResponseOptionsImpl.header(ValidatableResponseOptionsImpl.java:142)
at restasure.restaureapi.googleapi.test_ResponseHeaderData_ShouldBeCorrect(googleapi.java:39)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test_ResponseHeaderData_ShouldBeCorrect -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:restasure.restaureapi.googleapi@66a3ffec]" name="afterClass" is-config="true" duration-ms="2" started-at="2019-11-05T15:28:10Z" finished-at="2019-11-05T15:28:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- restasure.restaureapi.googleapi -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
